Kickstarter-iOS/Views/Cells/PledgeShippingLocationCell.swift

```diff
@@ -0,0 +1,105 @@
+import Library
+import Prelude
+import UIKit
+
+final class PledgeShippingLocationCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var adaptableStackView: UIStackView = { UIStackView(frame: .zero) }()
```

Comment:
What does adaptable mean here? Might help to name it based on what it contains for quicker reference.

Reply:
It adapts its axis based on the accessibility font size...for regular sizes it uses horizontal axes..for a11y sizes it uses vertical

Reply:
I've used to call it `inputStackView` but renamed it to this because it spans between multiple cells and always acts as the a11y adaptable container.